Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG DOCKER_BASE_IMAGE
+FROM $DOCKER_BASE_IMAGE
+ARG VCS_REF
+ARG BUILD_DATE
+LABEL \
+    maintainer="https://ocr-d.de/kontakt" \
+    org.label-schema.vcs-ref=$VCS_REF \
+    org.label-schema.vcs-url="https://github.com/qurator-spk/dinglehopper" \
+    org.label-schema.build-date=$BUILD_DATE
+
+WORKDIR /build/dinglehopper
+COPY pyproject.toml .
+COPY src/dinglehopper/ocrd-tool.json .
+COPY src ./src
+COPY requirements.txt .
+COPY README.md .
+COPY Makefile .
+RUN make install
+RUN rm -rf /build/dinglehopper
+
+WORKDIR /data
+VOLUME ["/data"]
```

Comment:
Actually, no, that's not a good idea. [Our spec](https://ocr-d.de/en/spec/docker#shell-entrypoint) recommends against it.

The reason is that it would only create more confusion: The command is scenario-specific, anyway. It can be `ocrd resmgr` stuff or any of the CLIs (usually more than one) in combination with different parameters. Or one of the server entrypoints.